trackbasket_be/resources/volunteer.py

```diff
@@ -5,33 +5,27 @@
 class Volunteers(Resource):
 
   def get(self,id):
-    volunteer = db.session.query(Volunteer).filter_by(volunteer_id=id).first()
+    volunteer = Volunteer.find_by_id(id)
```

Comment:
added the `find_by_id` method to the `volunteer`

Reply:
i like it!